infra/core/search/search-services.bicep

```diff
@@ -8,6 +8,7 @@ param sku object = {
 
 param authOptions object = {}
 param semanticSearch string = 'disabled'
+param authType string = 'rbac'
```

Comment:
Can you do a PR to update this? https://github.com/Azure/azure-dev/blob/main/templates/common/infra/bicep/core/search/search-services.bicep

Reply:
What updates are required? Do you just mean to pull from main?